### ISSUE TITLE
Outdated Documentation for deploying on AWS Elastic Beanstalk

### ISSUE THREAD

**Issue by zmunro** [metabase/metabase#32513]
### Describe the bug

The screenshots and instructions in the documentation for how to deploy metabase using AWS Elastic Beanstalk are out of date and don't match the current Elastic Beanstalk setup wizard making it hard to follow the prompts. The steps are out of order and do not accurately describe how to do a deploy. I am currently trying to use this document and having difficulty. 

### To Reproduce

1. Go to https://github.com/metabase/metabase/blob/master/docs/installation-and-operation/running-metabase-on-elastic-beanstalk.md
2. Go into Elastic Beanstalk in AWS
3. Try to follow the documentation from step 1 and see that the steps are out of order and screenshots are not accurate.

### Expected behavior

The screenshots should accurately depict the creation wizard. 

### Information about your Metabase installation

```JSON
N/A
```

### Severity

Low

**Comment by darksciencebase** [comment]
Related: #31151
